test/format/format_inline.h

```diff
@@ -281,11 +281,19 @@ static inline void
 wt_wrap_open_cursor(WT_SESSION *session, const char *uri, const char *config, WT_CURSOR **cursorp)
 {
     WT_DECL_RET;
+    bool is_checkpoint;
 
-    /* WT_SESSION.open_cursor can return EBUSY if concurrent with a metadata operation, retry. */
+    is_checkpoint = config != NULL && WT_PREFIX_MATCH(config, "checkpoint");
+
+    /*
+     * WT_SESSION.open_cursor can return EBUSY if concurrent with a metadata operation, retry. It
+     * can also return ENOENT, this is fine in the case of a checkpoint cursor.
+     */
     while ((ret = session->open_cursor(session, uri, NULL, config, cursorp)) == EBUSY)
         __wt_yield();
-    testutil_checkfmt(ret, "%s", uri);
+    testutil_assertfmt(ret == 0 || (is_checkpoint && ret == ENOENT), "%s", uri);
+    if (is_checkpoint && ret == ENOENT && *cursorp != NULL)
```

Comment:
Q Is this conditional necessary given the assertion above?
A quick look does not indicate to me that `testutil_assertfmt` can be defined empty.

Reply:
👍 Thank you for clearing that up for me.